packages/@expo/cli/src/export/exportApp.ts

```diff
@@ -60,13 +59,12 @@ export async function exportAppAsync(
   }
 
   const publicPath = path.resolve(projectRoot, env.EXPO_PUBLIC_FOLDER);
-
   const outputPath = path.resolve(projectRoot, outputDir);
-  const assetsPath = path.join(outputPath, 'assets');
-
-  await Promise.all([assetsPath].map(ensureDirectoryAsync));
 
-  await copyPublicFolderAsync(publicPath, outputPath);
+  // TODO: Remove when this is abstracted into the files map
+  if (!platforms.includes('web') || !useServerRendering) {
+    await copyPublicFolderAsync(publicPath, outputPath);
```

Comment:
This check would prevent copying public files when running `npx expo export` with web set to a server rendering output. We should probably just always copy the public folder since it doesn't contain private data.

Reply:
Ah, true, I forgot about that case. I'll just remove this check then. It's not strictly needed anyway